README.md

```diff
@@ -1 +1,245 @@
-# Conference-site-Capstone-Project-Module1
\ No newline at end of file
+# Conference-site-Capstone-Project-Module1
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
```

Comment:
- [Optional] To keep your code clean, I suggest you delete this commented code. 👍

Reply:
- [ OPTIONAL] As previous reviewer suggested to keep your code clean, I suggest you delete this commented code. 👍